es/criteria/continuous-integration.md

```diff
@@ -32,33 +32,33 @@ order: 12
 * Probar situaciones de la vida real.
 * Garantizar que el código ofrezca exactamente el mismo resultado de las políticas.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"